### DISCUSSION TITLE
An error occurred during installation. The virtual machine failed to start.

### DISCUSSION THREAD

**Discussion by MaximeLuk** [utmapp/UTM#5090] [UTM Help (macOS)]
I have an issue, when I make a macOS vm, it gives me this error message - An error occurred during installation. The virtual machine failed to start. - please help

Edit: this also happens on parallels

**Comment by osy** [comment]
Check that your IPSW is correct and restart your computer in case there's some weird macOS bug.

**Reply by MaximeLuk** [reply]
I got 3 ipsw and they were all wrong, but let me try restarting

**Reply by MaximeLuk** [reply]
Wow, I can't believe that restarting helped

**Reply by hryamzik** [reply]
In my case this doesn't help, maybe that's due to latest update (14.1.2).

**Reply by hryamzik** [reply]
OK, newer version of ipsw solved this.

**Reply by sbalk** [reply]
Hi,

I had the same problem on an M3, Sonoma 14.6.1 with UTM 4.5.3.

What didn't work:
- Using various IPSW files (14.1.1, 14.1.2, 14.4.1, 14.5, 14.6.1). They all gave me the same error message, namely the topic of this dicsussion.
- Restart and try again.
- Update UTM.

What worked:
- Use IPSW 12.6.1, couple of restarts and upgrade to 14.6.1 

For internet connection, I had to add a Network device in the guest settings and change Network mode to Bridged (Advanced).
